common/autoinstallers/rush-prettier/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "prettier": "2.3.1",
-    "pretty-quick": "~3.1.0"
+    "prettier": "3.0.0",
+    "lint-staged": "13.2.3"
```

Comment:
Is it changed because of prettier version update?

Reply:
Yes, pretty-quick does not work with this version. lint-staged is also more versatile